Resources/Prototypes/_LostParadise/Entities/Objects/Decoration/halloween.yml

```
@@ -31,4 +33,93 @@
+        sound:
+          collection: WoodDestroyHeavy
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+
+- type: entity
+  id: LPPPumpkinBigDeco
+  name: pumpkin
+  parent: BaseSign
+  description: Decorative pumpkins with brush marks on it
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)